MyCoreApp/Pages/Index.cshtml

```diff
@@ -5,6 +5,6 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Hello World!</h1>
+    <h1 class="display-4">Hello Haneef Muhammad!</h1>
```

Comment:
Exclamation mark not required.

Reply:
@dila-liyanage Have added the requested change.